sdk/scala-sdk/src/main/scala/kalix/scalasdk/valueentity/ValueEntity.scala

```diff
@@ -34,7 +34,7 @@ object ValueEntity {
 
       def updateState(newState: S): OnSuccessBuilder[S]
 
-      def deleteState: OnSuccessBuilder[S]
+      def deleteState(): OnSuccessBuilder[S]
```

Comment:
not sure how this will affect existing users, but it's a side effect function, so `()`

Reply:
Compiler warning (or possibly error) saying don't do non-param call to method with empty params, I think that is ok. 

(I'm fine with this change but It's not a side effect function though, it returns an effect saying "delete it", it doesn't actually do the delete)

Reply:
They will get a warning only, unless they are using a strict compiler configuration.

Not big issue, let's just do it.